File: backend/src/main/java/com/vtb/payandsave/response/ProfileResponse.java
package com.vtb.payandsave.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor
@Getter
@Setter
public class ProfileResponse {
    private Long account_id;
    private String phoneNumber;
    private String name;
    private String surname;
    private Long superPriorityTarget_id;
    private boolean isUseCashBack;
    private boolean isEvenDistribution;
    private boolean isPercentageOnBalance;
}

File: frontend/src/app/response/ProfileResponse.ts
export class ProfileResponse {
  name: string;
  surname: string;
  isUseCashBack: boolean;
  isEvenDistribution: boolean;
  isPercentageOnBalance: boolean;

  constructor(name: string, surname: string, isUseCashBack: boolean, isEvenDistribution:boolean, isPercentageOnBalance: boolean) {
    this.name = name;
    this.surname = surname;
    this.isUseCashBack = isUseCashBack;
    this.isEvenDistribution = isEvenDistribution;
    this.isPercentageOnBalance = isPercentageOnBalance;
  }
}
